process/peer/validatorsProvider.go

```diff
@@ -85,6 +85,21 @@ func (vp *validatorsProvider) GetLatestValidators() map[string]*state.ValidatorA
 	return mapToReturn
 }
 
+// GetLatestValidatorInfos gets the latest configuration of validators per shard from the peerAccountsTrie
+func (vp *validatorsProvider) GetLatestValidatorInfos() (map[uint32][]*state.ValidatorInfo, error) {
```

Comment:
Is this route protected? As this is costly as processing

Reply:
it will be used for updates in the cache in order to have the "latest" version of the validators: mainly the leaving that were processed by the metachain and will be removed in the next epoch.